pocs/core.py

```diff
@@ -1,7 +1,10 @@
+import os
+import sys
 import queue
 import time
 import zmq
 
+from warnings import warn
```

Comment:
Please move towards importing packages and modules, not the members of modules.

Reply:
Hm, I must have understood the conversation backwards. The `from` syntax should be the preferred. There is no pep8 specifically about this but the recommendation is to import the minimal needed from a package as opposed to the entre package or module.

Reply:
I'm not referencing pep8, but instead the Google Python Style Guide, which recommends importing packages or modules, but not the members of modules (e.g. not the functions and classes in a module). This is intended to make it easier for a reader to tell where a symbol comes from (i.e. if I see foo.Bar(), I know that the symbol Bar is defined in module foo, assuming that is a constructor call).